base_tier_validation_correction/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,10 @@
+This module provide a new operation to correct the information in tier.review for any document under tier validation.
+
+For example, a document started validation, a tier review is assigned to Mr. Smith.
+
+However, since Mr. Smith has urgent business oversea for a few days,
```

Comment:
Mr. Smith seems a busy guy, I pray for his family :stuck_out_tongue_closed_eyes:

Reply:
hehe, yes he is.